wsd/DocumentBroker.cpp

```diff
@@ -834,6 +835,12 @@ bool DocumentBroker::isAlive() const
     return _childProcess && _childProcess->isAlive();
 }
 
+void DocumentBroker::timeoutNotLoaded(const std::chrono::steady_clock::time_point& now)
+{
+    if (!_stop && !_poll->isAlive() && !isLoaded() && now - _createTime > std::chrono::seconds(60))
```

Comment:
I suggest we use `per_document.limit_load_secs`, which is already supported (in the thread). From the user's perspective, it's all part of loading, so we just need to start tracking time from an earlier point (`_createTime`). Although, again, we can fold `_threadStart` with `_createTime` and have a single one.

Reply:
limit_load_secs looks good, done. Not done is to replace _threadStart with _createTime and do all duration from construction time rather than pollThread time. Is that a change we want.

Maybe a local threadStart in DocumentBroker::pollThread, so that part doesn't change in behaviour, but _loadDuration gets derived from _createTime so is perhaps a little longer than before. and stats are shown from _createTime

Reply:
We can keep _createTime, not a major issue. Was just thinking of simplifying while also reviewing the logic overall. 

I think the original thinking was to delay the start of the limit_load_secs to be strictly the time spent loading the document (as opposed to getting internals ready). In practice, it shouldn't really matter much at all. And if we have to add a new timeout to track the "internals getting ready" bit, then it makes sense to me to unify them.